File: Frontend/src/app/dtos/productoDto.ts
export interface ProductoDto {
    idProducto: number;
    coProducto: string;
    nombre: string;
    precioCompra: number;
    precioVenta: number;
    porcentajeGanancia: number;
    descripcion: number;
    fechaVencimiento: Date;
    fechaRegistro: Date;
    fechaIngreso: Date;
    stock: number;
    marcaProducto: string;
    tipoProducto: string;
    imagen: string;
    estado: string;
}

File: Backend/dtodo-backend/src/main/java/xyz/pangosoft/dtodo/dto/ProductoDto.java
package xyz.pangosoft.dtodo.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProductoDto {
    private Integer idProducto;
    private String codProducto;
    private String nombre;
    private BigDecimal precioCompra;
    private BigDecimal precioVenta;
    private float porcentajeGanancia;
    private String descripcion;
    private Date fechaVencimiento;
    private Date fechaIngreso;
    private LocalDateTime fechaRegistro;
    private int stock;
    private String marcaProducto;
    private String tipoProducto;
    private String estado;
}
